src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -98,6 +98,8 @@ export default function FormNav(props) {
       <va-segmented-progress-bar total={chapters.length} current={current} />
       <div className="schemaform-chapter-progress">
         <div
+          role="progressbar"
+          aria-label={`Step ${current} of ${chapters.length}`}
           aria-valuenow={current}
           aria-valuemin="1"
           aria-valuetext={stepText}
```

Comment:
We may be able to delete all of this since it's built into the `<va-segmented-progress-bar>` right above.

[Related support thread](https://dsva.slack.com/archives/CBU0KDSB1/p1645031746860669)